roles/deploy/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: Initialize
   deploy_helper: path="{{ project_root }}" state=present
 
+- name: Update git remote URL
+  shell: "cd {{ project_source_path }} && git remote set-url origin {{ project_git_repo }} || :"
+  register: git_status
+
```

Comment:
I agree with @louim in this case about small tasks. It's also better when failures happen since you'll know it's a problem with a single command instead of a part of a complex command.